pkg/scheduler/scheduler.go

```diff
@@ -373,21 +386,27 @@ func (s *Scheduler) scheduleNext() bool {
 	}
 	defer s.queue.Done(key)
 
+	start := time.Now()
+
 	var err error
 	switch s.getScheduleType(key.(string)) {
 	case FirstSchedule:
 		err = s.scheduleOne(key.(string))
 		klog.Infof("Start scheduling binding(%s)", key.(string))
+		metrics.BindingSchedule("FirstSchedule", metrics.SinceInSeconds(start), err)
```

Comment:
Why not use the const `FirstSchedule`, like `string(FirstSchedule)`?

Reply:
ok